SessionMessagingKit/Utilities/Profile+CurrentUser.swift

```diff
@@ -105,10 +105,9 @@ public extension Profile {
                                 url: result.downloadUrl,
                                 key: result.encryptionKey,
                                 filePath: result.filePath,
-                                sessionProProof: sessionProProof
+                                sessionProProof: dependencies.mutate(cache: .libSession) { $0.getProProof() }
                             ),
                             profileUpdateTimestamp:  dependencies.dateNow.timeIntervalSince1970,
```

Comment:
Above you are using the `[cache: snodeAPI].currentOffsetTimestampMs()` so should probably update this one to be consistent